igor-web/src/test/groovy/com/netflix/spinnaker/igor/artifacts/TestArtifactService.java

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.igor.artifacts;
+
+import com.netflix.spinnaker.igor.model.ArtifactServiceProvider;
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
+import com.netflix.spinnaker.kork.web.exceptions.NotFoundException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class TestArtifactService implements ArtifactService {
```

Comment:
Can this be relocated to the java source set?

Reply:
Yes! Moving.